src/components/message-input/index.tsx

```diff
@@ -1,47 +1,88 @@
 import React from 'react';
-
+import { MentionsInput, Mention } from 'react-mentions';
 import classNames from 'classnames';
+import { userMentionsConfig } from './mentions-config';
+import { User } from '../../store/users';
 
 require('./styles.scss');
 
 export interface Properties {
   className?: string;
   placeholder?: string;
   isUserConnected?: boolean;
-  onSubmit: (message: string) => void;
+  onSubmit: (message: string, mentionedUsers: string[]) => void;
+  users: User[];
 }
 
 interface State {
   value: string;
+  userIds: string[];
 }
 
 export class MessageInput extends React.Component<Properties, State> {
-  state = { value: '' };
-  onSubmit = (e) => {
+  state = { value: '', userIds: [] };
+  onSubmit = (e): void => {
```

Comment:
you don't have to change this now, but in the future let's not add types to things like this. adding too many types makes it harder to make changes. for things like this typescript already knows what the return type is.